src/WeatherLibrary/Weather.cs

```diff
@@ -0,0 +1,174 @@
+﻿namespace WeatherLibrary;
+
+public class Weather
+{
+    static readonly HttpClient client = new HttpClient();
+    private static string APIkey = "7cc9500e6ce460aa0d0c8463c994b772";
+
+    public enum Language
+    {
+        af, 
```

Comment:
nit: по style guide'ам обычно с большой буквы
```suggestion
        Af, 
```

Reply:
nit: сам enum лучше в отдельном файле, тем более он длинный такой